torch/_inductor/compile_fx.py

```diff
@@ -241,6 +241,19 @@ def fake_tensor_prop(
     return fake_mode
 
 
+def get_mutating_use_stack_trace(placeholder_node) -> Optional[str]:
+    # reinplaced uses might have a single, non-copy_ use
+    if len(placeholder_node.users) == 1:
+        return next(placeholder_node.users).meta.get("stack_trace", None)
+
+    for use in placeholder_node.users:
+        if use.target == torch.ops.aten.copy_.default:
+            if stack_trace := use.meta.get("stack_trace", None):
+                return stack_trace
+
+    return None
```

Comment:
Is it possible to get here? (if not maybe we should just raise an assertion error). My understanding is that we only call this fn on placeholders that were mutated, which means that they should have a user.

Reply:
oh right lol didn't realize also we hit this if there is no stack trace (at first I read it as "we only get here if the placeholder has zero users")